src/styles.js

```diff
@@ -70,18 +65,26 @@ export default customOptions => {
     const hljsStyles = Object.keys(colors)
         .filter(k => ignoredColors.indexOf(k) === -1)
         .map(name => {
-            // convert from camelCase to hyphen-case
-            const type = name.replace(/[A-Z]/g, char => `-${char.toLowerCase()}`);
-            return `.${cssMap.container} .hljs-${type} { color: ${colors[name]}; }`;
-        }).join('\n')
-
-    insertCss(csjs.getCss(cssMap) + hljsStyles + `
-    .${cssMap.container} .hljs { background: ${colors.background}; }
-    .${cssMap.container} .hljs-emphasis { font-style: italic; }
-    .${cssMap.container} .hljs-strong { font-weight: bold; }
-    `)
+            const type = name
+                // convert from camelCase to hyphen-case
+                .replace(/[A-Z]/g, char => `-${char.toLowerCase()}`)
+                // replace stringed object keys to have no strings; i.e. {'cool-stuff': red; }
+                .replace(/['|"](.*)['|"]/g, '$1');
 
+            return `.${cssMap.container} .hljs-${type} { color: ${colors[name]}; }`;
+        }).join('\n');
 
+    // hljs overrides
+    insertCss(
+        csjs.getCss(cssMap) + hljsStyles + `
+            .${cssMap.container} .hljs {
+                background: ${colors.background};
+                color: ${colors.textColor};
```

Comment:
I wanted to avoid this, since it'll overwrite the default `color` of any imported hljs styles.

Reply:
Honestly forgot why I did this

Reply:
Remember now, I did this because the custom color wasn't coming thru